src/main/java/com/shopify/minishopify/controllers/ShopController.java

```diff
@@ -0,0 +1,63 @@
+package com.shopify.minishopify.controllers;
+
+import com.shopify.minishopify.model.Shop;
+import com.shopify.minishopify.repository.ShopRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("shops")
```

Comment:
base path is /api

Reply:
We can set that in the `application.properties` file.